x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/edit_package_policy_page/index.tsx

```diff
@@ -446,6 +446,7 @@ export const EditPackagePolicyForm = memo<{
         onChange={handleExtensionViewOnChange}
         validationResults={validationResults}
         isEditPage={true}
+        isAgentlessEnabled={true}
```

Comment:
Are these props being hardcoded because of a testing typo or is this component being set to check agentless elsewhere?

If it is not being used anywhere else we should consider removing those props.

Reply:
I do need the `isAgentlessEnabled` prop to show Agentless fields on the Edit Page which fixes the bug. As @maxcold above to the value shouldn't be hardcoded  but using `isAgentlessEnabled` flag.